sites/public/__tests__/components/applications/ApplicationsView.test.tsx

```diff
@@ -0,0 +1,386 @@
+
+    it("should show missing closed application when on the matching tab", async () => {
+      server.use(
+        rest.get("http://localhost:3100/applications/publicAppsView", (_req, res, ctx) => {
+          return res(ctx.json(getApplications(2, 0, 2, ApplicationsIndexEnum.closed)))
+        })
+      )
+
+      renderApplicationsView(ApplicationsIndexEnum.closed)
+      // Tab Panel
+      const allApplicationsTab = screen.getByTestId("total-applications-tab")
+      expect(await within(allApplicationsTab).findByText("4")).toBeInTheDocument()
+
+      const closedApplicationsTab = screen.getByTestId("closed-applications-tab")
+      expect(within(closedApplicationsTab).getByText("0")).toBeInTheDocument()
+
+      const openApplicationsTab = screen.getByTestId("open-applications-tab")
+      expect(within(openApplicationsTab).getByText("2")).toBeInTheDocument()
+
+      const lotteryTab = screen.getByTestId("lottery-runs-tab")
+      expect(within(lotteryTab).getByText("2")).toBeInTheDocument()
+
+      expect(
+        await screen.findByText(
+          "The listings you've applied to are either still accepting applications or already have lottery results posted."
+        )
+      ).toBeInTheDocument()
+    })
+
+    it("should show missing open application when on the matching tab", async () => {
+      server.use(
+        rest.get("http://localhost:3100/applications/publicAppsView", (_req, res, ctx) => {
+          return res(ctx.json(getApplications(0, 2, 2, ApplicationsIndexEnum.open)))
+        })
+      )
+
+      renderApplicationsView(ApplicationsIndexEnum.open)
+      // Tab Panel
+      const allApplicationsTab = screen.getByTestId("total-applications-tab")
+      expect(await within(allApplicationsTab).findByText("4")).toBeInTheDocument()
+
+      const closedApplicationsTab = screen.getByTestId("closed-applications-tab")
+      expect(within(closedApplicationsTab).getByText("2")).toBeInTheDocument()
+
+      const openApplicationsTab = screen.getByTestId("open-applications-tab")
+      expect(within(openApplicationsTab).getByText("0")).toBeInTheDocument()
+
+      const lotteryTab = screen.getByTestId("lottery-runs-tab")
+      expect(within(lotteryTab).getByText("2")).toBeInTheDocument()
+
+      expect(
+        await screen.findByText(
+          "None of the listings you've applied to are still accepting applications."
+        )
+      ).toBeInTheDocument()
+    })
+  })
+
+  it("should show the page with only proper applications count", async () => {
+    server.use(
+      rest.get("http://localhost:3100/applications/publicAppsView", (_req, res, ctx) => {
+        return res(ctx.json(getApplications(1, 2, 3)))
+      })
+    )
+    renderApplicationsView(ApplicationsIndexEnum.open)
+
+    expect(
+      screen.queryByText("It looks like you haven't applied to any listings yet.")
+    ).not.toBeInTheDocument()
+
+    // TODO: in core we an "view application" for closed listings but not in Doorway. Is that intentional
```

Comment:
@ColinBuyck @emilyjablonski (I can't tag Sarah right now, but also consider her tagged her) When pulling over this test I found that Doorway has a slightly different behavior that core. I want to get verification that this is intentional 

Scenario:
* User have applied to multiple listings that are in various statuses. At least one of them is a closed listing
* go to your dashboard and you will see all of the applications, however, you won't see the link to click on the application for any of the closed listings 

This is because in `StatusItemWrapper.tsx` the applicationUrl is not set if `props.application?.listings?.status === ListingsStatusEnum.closed`. That functionality doesn't exist in core. Is that correct?

Reply:
Yep! This [ticket](https://github.com/bloom-housing/bloom/issues/4053) says "In Doorway only, when a status is set to “Applications closed”, only one button should be visible: 1) See listing. The application is no longer visible to the user. In Core, the user should still be able to view their application."